.github/workflows/lint.yml

```diff
@@ -0,0 +1,24 @@
+name: Lint
+
+on:
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build:
+    name: Lint
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+        with:
+          # Ensure full list of changed files within `super-linter`
+          fetch-depth: 0
+      - name: Run linters
+        uses: github/super-linter@v4
+        env:
+          DEFAULT_BRANCH: ${{ github.base_ref }}
+          EDITORCONFIG_FILE_NAME: /src/.editorconfig
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Why do we need to feed in GITHUB_TOKEN?

Reply:
For check suite in case we add shell, powershell, etc linters

<img width="1050" alt="Screen Shot 2022-10-18 at 11 43 10 AM" src="https://user-images.githubusercontent.com/13227161/196478789-d4f43cc9-e339-41b4-a9b4-922f24f21a81.png">

Reply:
Can we not have this since PR from fork repo is not going to work with it?

Reply:
removed